runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaOverlay.java

```diff
@@ -70,6 +71,13 @@ public Dimension render(Graphics2D graphics)
 			return null;
 		}
 
+		final Player localPlayer = client.getLocalPlayer();
+		if (localPlayer == null) return null;
```

Comment:
The player is never null in overlays so this is safe to omit

Reply:
Thanks for the clarification on the null safety. I'll make this change.